dependabot: Try to receive only security updates

ashie opened the pull request:
We don't intend to receive all updates, we want only security updates.
For this purpose, `open-pull-requests-limit: 0` seems effective because it doesn't affect to security updates:

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#open-pull-requests-limit

kenhys reviewed (commented):
Using `open-pull-requests-limit: 0` is good approach. :+1: 

But does `directory: "/docker-image" work as intended?
(I recall that it caused an error before)

ashie commented:
> (I recall that it caused an error before)

What kind of error was it?

ashie commented:
Ah, dependabot see only Gemfile just under the directory?

ashie committed:
dependabot: Try to receive only security updates

We don't intend to receive all updates, we want only security updates.
For this purpose, `open-pull-requests-limit: 0` seems effective because
it doesn't affect to security updates:

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#open-pull-requests-limit

Signed-off-by: Takuro Ashie <ashie@clear-code.com>

ashie commented:
Finding it recursive doesn't seem supported yet: https://github.com/dependabot/dependabot-core/issues/2178

ashie commented:
Done at #1081
